3pi-serial-slave/slave.cpp

```diff
@@ -520,6 +520,8 @@ void DriveStraight(PID_t *pid)
 void DriveStraightDistance(PID_t *left_pid, PID_t *right_pid)
 {
     //get speed from mbed
+    //speed typically set as 40 for a smooth move
```

Comment:
Let's remove the comment of "get speed from mbed" and put something like "recommended starting speed is 40 for irisbots".

Reply:
yes i did